src/client-websocket.ts

```diff
@@ -0,0 +1,11 @@
+declare let window: any;
+let _global = typeof global !== 'undefined' ? global : (typeof window !== 'undefined' ? window: {});
+
+import * as NodeSocket from 'ws';
```

Comment:
The previous package, `websocket` had the logic that inside the file `client-websocket` internally, and this was the `browser` file. 

I changed the usage so `ws` will be required only when there is no native WebSocket available, as suggested here: https://github.com/websockets/ws/issues/837 , and also added `browser` property to expose `./dist/client.js` only.